README.md

```diff
@@ -1,18 +1,24 @@
 # weave-gitops
 
-Weave GitOps
+Weave GitOps is a simple open source developer platform for people who want cloud native applications, without needing Kubernetes expertise.  Start with what developers need to run apps, and then easily extend to define and run your own enterprise platform.
```

Comment:
_puts on marketing hat_ Needs a mention of GitOps here. We aren't really targeting just any K8s users, they need to want to use GitOps.

Maybe even a mention of Flux?

Reply:
Flux is mentioned near the bottom of the blurb.